netfields/forms.py

```diff
@@ -1,21 +1,24 @@
-from ipaddress import ip_interface, ip_network, _IPAddressBase, _BaseNetwork
+from ipaddress import _IPAddressBase, _BaseNetwork, AddressValueError
 from netaddr import EUI, AddrFormatError
 
 from django import forms
 from django.core.exceptions import ValidationError
 
 from netfields.compat import text_type
 from netfields.mac import mac_unix_common
+from netfields.address_families import BOTH_FAMILIES, get_interface_type_by_address_family, get_network_type_by_address_family
 
 
 class InetAddressFormField(forms.Field):
     widget = forms.TextInput
     default_error_messages = {
         'invalid': u'Enter a valid IP address.',
+        'invalid_address_family': u'Enter a valid %(address_family)s address.',
```

Comment:
This is going to produce errors like `Enter a valid 4 address` or `Enter a valid None address`

Reply:
Changed it in `to_python`, not that this is only used when `address_family` is specified.